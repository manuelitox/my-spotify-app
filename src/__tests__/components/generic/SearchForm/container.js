import React from 'react'
import { shallow } from 'enzyme'

import SearchFormContainer from 'components/generic/SearchForm/container'

describe('Component | SearchFormContainer', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <SearchFormContainer />
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})