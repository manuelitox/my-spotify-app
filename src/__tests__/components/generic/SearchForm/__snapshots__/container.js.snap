// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | SearchFormContainer should render a component 1`] = `
<SearchForm
  onChange={[Function]}
/>
`;
